[18.0][MIG] stock_cycle_count: Migration to 18.0

xaviedoanhduy opened the pull request:
includes some bug fixes:

- https://github.com/OCA/stock-logistics-warehouse/pull/2394
- https://github.com/OCA/stock-logistics-warehouse/pull/2395

reports:

- [Cycle Count.pdf](https://github.com/user-attachments/files/21461078/Cycle.Count.pdf)
- [Accuracy Report.pdf](https://github.com/user-attachments/files/21461244/Accuracy.Report.pdf)


note: 

- Because the Accuracy Report uses `web.external_layout` and introduce the change from `https://github.com/odoo/odoo/pull/169512`, change the table showing the location details in this report, based on another report, the Purchase Order report, which also `uses web.external_layout`

old table:
<img width="820" height="875" alt="image" src="https://github.com/user-attachments/assets/ed591e3e-3fdc-41f2-8e05-37780fcac823" />

new table:
<img width="815" height="874" alt="image" src="https://github.com/user-attachments/assets/b6234543-bfe6-4263-ab22-9e9272920d69" />

LoisRForgeFlow committed:
[ADD] stock_cycle_count

MiquelRForgeFlow committed:
[ADD] Adding tests

LoisRForgeFlow committed:
[IMP] stock_cycle_count:
 * Update README.
 * Fixes: DEFAULT_SERVER_DATETIME_FORMAT, PERCENT variables and sale price calculation.

LoisRForgeFlow committed:
[IMP] tests.

LoisRForgeFlow committed:
[9.0][FIX] Solves issue when doing picking with several moves.

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: add the possibility to define rules for specific zones.

LoisRForgeFlow committed:
[9.0][FIX] stock_cycle_count: Do not propose cycle counts in the past.

LoisRForgeFlow committed:
[9.0][FIX] stock_cycle_count: Fix a bug when combining automatic and manual creation of cycle counts.

LoisRForgeFlow committed:
[9.0][FIX] stock_cycle_count: fields only editable in draft and add tracking to some of them.

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: Enhance tree view with filter and colors.

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: The configuration of an inventory adjustment related to a cycle count cannot be modified.

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: update existing cycle counts instead of cancel them.

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: add server action to confirm several CC at a time

oca-transbot committed:
OCA Transbot updated translations from Transifex

MiquelRForgeFlow committed:
[FIX] stock_cycle_count (usage cannot be 'view')

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[10.0][MIG] stock_cycle_count

LoisRForgeFlow committed:
cycle_count: fix cycle count state issue

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: fix accuracy computation and store it

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: basic_accuracy report

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: enhance views

LoisRForgeFlow committed:
[9.0][IMP] stock_cycle_count: cycle counts report

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[10.0][IMP] fix api issues and clean up code

LoisRForgeFlow committed:
[10.0] stock_cycle_count: adapt to follow OCA convention.

LoisRForgeFlow committed:
[10.0][IMP] stock_cycle_count:
* simplify fetching latest inventory date
* use api.depends where needed
* add hook for cycle count creation
* take advantage of api.multi on check_zero_confirmation method

LoisRForgeFlow committed:
[10.0][UPD] stock_cycle_count: add roadmap.

sergio-teruel committed:
[ADD] stock_secondary_unit: New module for getting stock in a secondary unit

LoisRForgeFlow committed:
[11.0][MIG] stock_cycle_count

JordiBForgeFlow committed:
stock_cycle_count: fix issue with nan while doing the mean calculation

pedrobaeza committed:
[FIX] stock_cycle_count: Don't lose return value

For avoiding an error validating pickings.

sergio-teruel committed:
[11.0][FIX] stock_secondary_unit: Remove procurement_rule move. Fix dependencies

kittiu committed:
[12.0][MIG] stock_secondary_unit

sergio-teruel committed:
[FIX] stock_secondary_unit: Fix key error in vals

Commit:
[12.0][MIG] stock_cycle_count: Migration to 12.0

LoisRForgeFlow committed:
[12.0][FIX/MIG] stock_cycle_count: fix zero confirmation rules.
Now zero-qty quants are not removed straight away but by a cron job.
Therefore, we have to also check that the quants are not zero-qty ones.

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 40.2% (51 of 127 strings)

Translation: stock-logistics-warehouse-12.0/stock-logistics-warehouse-12.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_cycle_count/zh_CN/

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (127 of 127 strings)

Translation: stock-logistics-warehouse-12.0/stock-logistics-warehouse-12.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_cycle_count/zh_CN/

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

digitalsatori committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (15 of 15 strings)

Translation: stock-logistics-warehouse-12.0/stock-logistics-warehouse-12.0-stock_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_secondary_unit/zh_CN/

enriquemartin committed:
[12.0][FIX] stock_secondary_unit: move line create round
On move line creation, the precision of secondary qty rounding is wrong; the unit of measure factor is being used instead of rounding.

sergio-teruel committed:
[IMP] stock_cycle_count: Create test user without password to pass tests with other modules like password_policy

ps-tubtim committed:
[IMP] stock_secondary_unit: black, isort

ps-tubtim committed:
[MIG] stock_secondary_unit: Migration to 13.0

jimhoefnagels committed:
[IMP] stock_cycle_count: black, isort

jimhoefnagels committed:
[MIG] stock_cycle_count: Migration to 13.0

OCA-git-bot committed:
stock_cycle_count 13.0.1.0.1

sergio-teruel committed:
[IMP] stock_secondary_unit: Display info only for inventory users

MiquelRForgeFlow committed:
[UPD] Eficent -> ForgeFlow

sergio-teruel committed:
[IMP] stock_secondary_unit: Remove unnecessary code. Always compute as field.
TT26606

sergio-teruel committed:
[FIX] stock_secondary_unit: Incorrect secondary unit link in product variants view on common form. Create related template secondary unit field.
TT27057

sergio-teruel committed:
[IMP] stock_secondary_unit: Use secondary units directly in pickings
[IMP] stock_secondary_unit: Use product_secondary_unit mixin model from product-attribute repository.

LoisRForgeFlow committed:
[IMP] stock_cycle_count: auto link inventories to planned cycle counts

simahawk committed:
Add stock_vertical_lift_qty_by_packaging

OCA-git-bot committed:
stock_cycle_count 13.0.1.1.0

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

oca-travis committed:
[UPD] Update stock_vertical_lift_qty_by_packaging.pot

sergio-teruel committed:
[FIX] stock_secondary_unit: Allow use secondary units in form view for mobile views
TT30538

celm1990 committed:
FIX: correct field name, location_ids instead of location_id

MateuGForgeFlow committed:
[14.0][MIG] stock_cycle_count: black, isort, prettier

MateuGForgeFlow committed:
[14.0][MIG] stock_cycle_count: Migration to 14.0

Commit:
[MIG] stock_vertical_lift_qty_by_packaging: Migration to 14.0

alan196 committed:
[IMP] stock_secondary_unit: black, isort, prettier

alan196 committed:
[MIG] stock_secondary_unit: migration to V14.

oca-travis committed:
[UPD] Update stock_vertical_lift_qty_by_packaging.pot

OCA-git-bot committed:
[UPD] README.rst

MateuGForgeFlow committed:
[FIX] stock_cycle_count: fix deprecation warnings

OCA-git-bot committed:
stock_cycle_count 14.0.1.1.1

florentx committed:
[FIX] adapt syntax for act_window lookup

OCA-git-bot committed:
stock_cycle_count 14.0.1.1.2

MateuGForgeFlow committed:
[14.0][IMP] stock_cycle_count: check modify

OCA-git-bot committed:
stock_cycle_count 14.0.1.2.0

CarlosRoca13 committed:
[MIG] stock_secondary_unit: Migration to 15.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 54.3% (69 of 127 strings)

Translation: stock-logistics-warehouse-14.0/stock-logistics-warehouse-14.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-14-0/stock-logistics-warehouse-14-0-stock_cycle_count/it/

rousseldenis committed:
[FIX] Licensing issues

rousseldenis committed:
[FIX] Development statuses

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
stock_vertical_lift_qty_by_packaging 14.0.1.0.1

OCA-git-bot committed:
stock_cycle_count 14.0.1.2.1

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (127 of 127 strings)

Translation: stock-logistics-warehouse-14.0/stock-logistics-warehouse-14.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-14-0/stock-logistics-warehouse-14-0-stock_cycle_count/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (127 of 127 strings)

Translation: stock-logistics-warehouse-14.0/stock-logistics-warehouse-14.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-14-0/stock-logistics-warehouse-14-0-stock_cycle_count/it/

JordiBForgeFlow committed:
[IMP] stock_cycle_count: Implement constrains in inventory adjustment.
The previous method was too restrictive, and would cause conflicts
in cases where another module would introduce changes to the inventory
adjustment that were not really affecting to the cycle count.

We implement the minimum necessary constrains to ensure consistency between
the cycle count and the related inventory adjusment.

DavidJForgeFlow committed:
[14.0][IMP] stock_cycle_count: add auto start adjustment option

OCA-git-bot committed:
stock_cycle_count 14.0.1.2.2

OCA-git-bot committed:
stock_cycle_count 14.0.1.3.0

DavidJForgeFlow committed:
[14.0][IMP] stock_cycle_count: inherit to stock.inventory

OCA-git-bot committed:
stock_cycle_count 14.0.1.4.0

Chanakya-OSI committed:
[15.0][MIG]stock_cycle_count

sergio-teruel committed:
[IMP] stock_secondary_unit: Compatibility with product variants
TT40323

lmignon committed:
[ADD] stock_picking_volume_packaging

This addon takes care of product packaging into the computation of move's volume

sergio-teruel committed:
[FIX] stock_secondary_unit: Remove _merge_moves_fields method. Now Odoo compute the change

Commit:
[UPD] Update stock_picking_volume_packaging.pot

OCA-git-bot committed:
[UPD] README.rst

AaronHForgeFlow committed:
[IMP] stock_inventory: do not delete inv adjustment groups if in process or done

mt-software-de committed:
[IMP] stock_picking_volume: Move _get_volume_for_qty to product.product

sergio-teruel committed:
[FIX] stock_secondary_unit: Propagate secondary_uom_id for new extra moves

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: stock-logistics-warehouse-15.0/stock-logistics-warehouse-15.0-stock_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-15-0/stock-logistics-warehouse-15-0-stock_secondary_unit/es/

n17 committed:
Added translation using Weblate (Finnish)

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

josep-tecnativa committed:
[IMP] stock_secondary_unit: test performance improvement
- Include context keys for avoiding mail operations overhead.

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_picking_volume_packaging
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_picking_volume_packaging/it/

sergio-teruel committed:
[FIX] stock_secondary_unit: The second qty is computed when a sml is created. Second qty compute the qty_done field.
The second uom not locked for non draft move lines

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-warehouse-14.0/stock-logistics-warehouse-14.0-stock_vertical_lift_qty_by_packaging
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-14-0/stock-logistics-warehouse-14-0-stock_vertical_lift_qty_by_packaging/it/

MarinaAForgeFlow committed:
[ADD] stock_move_purchase_uom

sergio-teruel committed:
[IMP] stock_secondary_unit: Make readonly secondary qty field in some states

Commit:
[UPD] Update stock_picking_volume_packaging.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_picking_volume_packaging
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_picking_volume_packaging/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_picking_volume_packaging
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_picking_volume_packaging/it/

lk-eska committed:
[MIG] stock_secondary_unit: Migration to 16.0

MarinaAForgeFlow committed:
[ADD] stock_move_location_purchase_uom

Commit:
[UPD] Update stock_move_purchase_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

ArnauCForgeFlow committed:
[15.0][IMP] stock_cycle_count: add auto complete into cron

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: deleted stock_inventory_exclude_sublocation dependency

OCA-git-bot committed:
[BOT] post-merge updates

JoanSForgeFlow committed:
[ADD] stock_cycle_count: line_accuracy in stock move lines

Commit:
[UPD] Update stock_move_location_purchase_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 86.1% (124 of 144 strings)

Translation: stock-logistics-warehouse-15.0/stock-logistics-warehouse-15.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-15-0/stock-logistics-warehouse-15-0-stock_cycle_count/it/

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: improved inventory_accuracy calculation

JoanSForgeFlow committed:
[FIX] stock_move_purchase_uom: test typo

OCA-git-bot committed:
[BOT] post-merge updates

JoanSForgeFlow committed:
[FIX] stock_cyle_count: remove duplicated field location_ids in tree view

mymage committed:
Translated using Weblate (Italian)

Currently translated at 86.1% (124 of 144 strings)

Translation: stock-logistics-warehouse-15.0/stock-logistics-warehouse-15.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-15-0/stock-logistics-warehouse-15-0-stock_cycle_count/it/

ArnauCForgeFlow committed:
[MIG] stock_cycle_count: Migration to 16.0

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: start inv adjustments depending on setting

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: preformance improvements

OCA-git-bot committed:
[BOT] post-merge updates

pilarvargas-tecnativa committed:
[IMP] stock_secondary_unit: make the secondary unit and secondary quantity fields optional

TT49686

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (144 of 144 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_cycle_count/it/

ArnauCForgeFlow committed:
[FIX] stock_cycle_count: prevent creating infinte cycle counts

ArnauCForgeFlow committed:
[FIX] stock_cycle_count: removed some asserts for AccessError

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (144 of 144 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_cycle_count/it/

ArnauCForgeFlow committed:
[FIX] stock_cycle_count: prefill_counted_quantity zero, counted, false

ArnauCForgeFlow committed:
[FIX] stock_cycle_count: inventory adjustment already adds these fields

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: added more tests

JoanSForgeFlow committed:
[IMP] stock_cycle_count: Enable edit and cascade updates for responsible_id

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (144 of 144 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_cycle_count/it/

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: added new group by required date

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: add multicompany rule

JoanSForgeFlow committed:
[IMP] stock_cycle_count: remove auto confirmation logic from the cron

ArnauCForgeFlow committed:
[FIX] stock_cycle_count: fixed tests since having initial quantity 0 isnt possible

Murtaza-SerpentCS committed:
[17.0][MIG] stock_picking_volume_packaging.

Commit:
[UPD] Update stock_picking_volume_packaging.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_secondary_unit/it/

ced-adhoc committed:
[16.0][FIX] stock_secondary_unit: use @api.model_create_multi in create method

JoanSForgeFlow committed:
[IMP] stock_cycle_count: Refactor responsible_id sync using compute/inverse fields

sergio-teruel committed:
[FIX] stock_secondary_unit: Error when user try to select a secondary uom without product in sale order line
TT51683

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: Remove useless filtered

JoanSForgeFlow committed:
[FIX] stock_cycle_count: use company of the location when creating cycle_count

ArnauCForgeFlow committed:
[IMP] stock_cycle_count: add rec company_id to domain

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (148 of 148 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_cycle_count/it/

rov-adhoc committed:
[IMP] stock_secondary_unit: pre-commit execution

rov-adhoc committed:
[MIG] stock_secondary_unit: Migration to 17.0

thomaspaulb committed:
[ADD] stock_inventory: index on stock.move.line for inventory_adjustment_id

In systems with many move lines, finding the move lines associated with a certain
inventory adjustment can become very slow. This index assists.

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: stock-logistics-warehouse-17.0/stock-logistics-warehouse-17.0-stock_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-17-0/stock-logistics-warehouse-17-0-stock_secondary_unit/it/

Commit:
[MIG] stock_secondary_unit: Migration to 18.0

arnaudlayec committed:
Translated using Weblate (French)

Currently translated at 100.0% (148 of 148 strings)

Translation: stock-logistics-warehouse-16.0/stock-logistics-warehouse-16.0-stock_cycle_count
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-16-0/stock-logistics-warehouse-16-0-stock_cycle_count/fr/

rousseldenis committed:
[FIX] stock_cycle_count: Don't use flush() anymore in tests

rousseldenis committed:
[FIX] stock_cycle_count: Don't use invalidate_cache() anymore in tests

OCA-git-bot committed:
[BOT] post-merge updates

rlimaeco committed:
[IMP] stock_vertical_lift_qty_by_packaging: pre-commit auto fixes

rlimaeco committed:
[UPD] Add div hideable to base view stock_vertical_lift

rlimaeco committed:
[MIG] stock_vertical_lift_qty_by_packaging: Migration to 18.0

LoisRForgeFlow committed:
[IMP] stock_warehouse_calendar: support push rules

mmequignon committed:
[MIG] stock_picking_volume_packaging: Migration to 18.0

xaviedoanhduy committed:
[IMP] stock_cycle_count: pre-commit auto fixes

xaviedoanhduy committed:
[FIX] stock_cycle_count: filter stock.inventory is not inherited yet

xaviedoanhduy committed:
[MIG] stock_cycle_count: Migration to 18.0

xaviedoanhduy committed:
[FIX] stock_cycle_count: fix multiple issues in location accuracy report

LoisRForgeFlow commented:
Hi @xaviedoanhduy 

Thanks for your work. Why don't you open this PR directly at OCA repo?

xaviedoanhduy commented:
Hi @LoisRForgeFlow, thank you for your feedback!

This PR is currently going through our internal review process before being proposed directly to the OCA repository.

Once it's been validated internally, we plan to open the official PR on OCA.

Appreciate your patience and stay tuned!

thienvh332 commented on stock_cycle_count/reports/report_stock_location_accuracy.py:
```diff
@@ -28,9 +28,10 @@ def _get_location_data(self, locations):
-            loc_counts = counts.filtered(lambda c: c.location_id == loc)
-            data[loc] = loc_counts
-        return data
+            location_data[loc] = counts.filtered(
+                lambda count, loc=loc: loc in count.location_ids
+            )
+        return location_data
```
What change leads to this change in logic?

xaviedoanhduy replied:
thanks, resolve conflicts from the painful rebase phase

thienvh332 commented on stock_cycle_count/models/stock_quant.py:
```diff
@@ -7,38 +7,38 @@
 class StockQuant(models.Model):
     _inherit = "stock.quant"
 
+    # pylint: disable=W8110
```
Using this approach gives the impression that you're hiding something. Although the outcome is the same, we should prioritize clarity to avoid raising suspicion from others.

xaviedoanhduy replied:
The bypass syntax that avoids this check is quite popular because it simply checks for the absence of a return when using `super()`, but in reality it is not perfect because there are many native functions that do not return anything. There are some programmers like me who did not know about this before and used return to bypass this check, but now thanks to this knowledge we should be clear whether this overridden function really needs to return or not to avoid confusing others

thienvh332 reviewed (approved):
LGTM

OCA-git-bot committed:
Merge PR #2382 into 18.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_picking_volume_packaging.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2409 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update stock_inventory.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-warehouse-18.0/stock-logistics-warehouse-18.0-stock_inventory
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-18-0/stock-logistics-warehouse-18-0-stock_inventory/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (80 of 80 strings)

Translation: stock-logistics-warehouse-18.0/stock-logistics-warehouse-18.0-stock_inventory
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-18-0/stock-logistics-warehouse-18-0-stock_inventory/it/

ArnauCForgeFlow committed:
[IMP] stock_move_location_purchase_uom: pre-commit auto fixes

ArnauCForgeFlow committed:
[IMP] stock_move_purchase_uom: pre-commit auto fixes

ArnauCForgeFlow committed:
[MIG] stock_move_purchase_uom: Migration to 18.0

ArnauCForgeFlow committed:
[MIG] stock_move_location_purchase_uom: Migration to 18.0

OCA-git-bot committed:
Merge PR #2416 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update stock_move_purchase_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-warehouse-18.0/stock-logistics-warehouse-18.0-stock_move_purchase_uom
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-18-0/stock-logistics-warehouse-18-0-stock_move_purchase_uom/it/

bo-eska committed:
Translated using Weblate (Turkish)

Currently translated at 98.7% (79 of 80 strings)

Translation: stock-logistics-warehouse-18.0/stock-logistics-warehouse-18.0-stock_inventory
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-18-0/stock-logistics-warehouse-18-0-stock_inventory/tr/

OCA-git-bot committed:
Merge PR #2417 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update stock_move_location_purchase_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

OCA-git-bot committed:
Merge PR #2280 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update stock_secondary_unit.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: stock-logistics-warehouse-18.0/stock-logistics-warehouse-18.0-stock_move_location_purchase_uom
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-18-0/stock-logistics-warehouse-18-0-stock_move_location_purchase_uom/it/

OCA-git-bot committed:
Merge PR #2385 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

jprokop1 committed:
Added translation using Weblate (Slovak)

florentx committed:
[FIX] stock_vertical_lift_qty_by_packaging: display Quantity to Pick on the bottom on the screen

florentx committed:
stock_vertical_lift: readability

florentx committed:
[FIX] stock_vertical_lift: ValueError on operation Save when move is split

OCA-git-bot committed:
Merge PR #2428 into 18.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2374 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update stock_vertical_lift_qty_by_packaging.pot

OCA-git-bot committed:
[BOT] post-merge updates

florentx committed:
stock_vertical_lift: port OCA/stock-logistics-shopfloor#51

OCA-git-bot committed:
Merge PR #2429 into 18.0

Signed-off-by sebalix

OCA-git-bot committed:
[BOT] post-merge updates
